editor/editor_settings.cpp

```diff
@@ -467,6 +467,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 
 	// Help
 	_initial_set("text_editor/help/show_help_index", true);
+    	_initial_set("text_editor/help/help_source_font_size", 14);
```

Comment:
If we set it here, the `EDITOR_DEF` in `editor_fonts.cpp` should be changed to `EDITOR_GET` to avoid defining it twice (with potentially different values if any is changed).

Reply:
I deleted:
"_initial_set("text_editor/help/help_source_font_size", 14);"
so help_source_font_size is not defined twice

Reply:
@akien-mga do you know what's the problem with the "continuous-integration/travis-ci/pr" check? I indented it but still don't pass the test

Reply:
You're using spaces in your indentation, see my comment above.

Reply:
IMO you should have kept the added `_initial_set` and removed the occurrence in `editor_fonts.cpp` (replacing it by `EDITOR_GET`).

It's weird to have a `hints` entry in `editor/editor_settings.cpp` that doesn't match any property defined in the same scope. And it also looks unsafe, as it's not obvious which one will be processed first.